R/remotes.R

```diff
@@ -176,6 +232,12 @@ renv_remotes_parse <- function(entry) {
     return(parsed)
   }
 
+  parsed <- catch(renv_remotes_parse_git(entry))
+  if (!inherits(parsed, "error")) {
+    parsed$type <- parsed$type %||% "git"
+    return(parsed)
+  }
+
```

Comment:
Upon reflection - https://github.com/rstudio/renv/pull/762/files#r644995771 seems redundant given this block

Reply:
Wrong - `renv_remotes_parse_finalize` will default anything to `"github"` before `parsed$type <- parsed$type %||% "git"`  can default it to `"git"`